app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -221,7 +221,9 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
       return true;
     } else if(($scope.currentTab == "hawkular" && $scope.emsCommonModel.ems_controller == "ems_container") &&
       ($scope.emsCommonModel.emstype) &&
-      ($scope.emsCommonModel.hawkular_hostname != '' && $scope.emsCommonModel.hawkular_api_port)) {
+      ($scope.emsCommonModel.hawkular_hostname != '' && $scope.emsCommonModel.hawkular_api_port) &&
```

Comment:
yes, but ... this is part of a big `else_if` block and this is the coding style here, I do not want to change it just for this line ... and I do not want to change the whole file :-)